utils/users.js

```diff
@@ -71,9 +71,11 @@ const getParticipantUserIds = async (participantArray) => {
 
     const promises = participantArray.map(async (participant) => {
```

Comment:
What if we use filter over here directly, since getUserId will return an id(positive value) or `false`?

Reply:
But the async function returns a Promise rather than a Boolean value. So we cannot directly replace map to filter